examples/server.rs

```diff
@@ -22,17 +22,22 @@ fn hello(
 }
 
 fn run() -> io::Result<()> {
-    let svr = hyperlocal::server::Http::new().bind("test.sock", || service_fn(hello))?;
+    match fs::remove_file("test.sock") {
```

Comment:
Nice. I'm been trying come up with a way to impl drop for servers that cleans up the created file. This has been an annoyance when running the example. Thanks for this!